db/migrate/20170714144125_add_deb.rb

```diff
@@ -0,0 +1,34 @@
+class AddDeb < ActiveRecord::Migration
+  def up
+    create_table "katello_debs" do |t|
+      t.string "uuid", :null => false, :limit => 255
+      t.timestamps
+      t.string 'name', :limit => 255
+      t.string 'version', :limit => 255
+      t.string 'architecture', :limit => 255
+      t.string 'filename', :limit => 255
+      t.string 'checksum', :limit => 255
+      t.string 'version_sortable', :limit => 255
+      t.string 'description', :limit => 255
```

Comment:
Reading https://www.debian.org/doc/debian-policy/#version  it looks like there could be value in a 'release' field, does pulp provide that?

Reply:
Do you mean 'revision'? Then we should also have epoch implemented. Right now, i think the version is imported as a whole into the `version_sorted` field. Is there a benefit in parsing that into three variables?

Reply:
trying to think of a good argument against this way, but i can't really, so i guess its fine for now :) 👍